core/src/test/scala/unit/kafka/admin/AddPartitionsTest.scala

```diff
@@ -137,12 +137,12 @@ class AddPartitionsTest extends BaseRequestTest {
     adminZkClient.addPartitions(topic3, topic3Assignment, adminZkClient.getBrokerMetadatas(), 7)
 
     // read metadata from a broker and verify the new topic partitions exist
-    TestUtils.waitUntilMetadataIsPropagated(servers, topic3, 1)
-    TestUtils.waitUntilMetadataIsPropagated(servers, topic3, 2)
-    TestUtils.waitUntilMetadataIsPropagated(servers, topic3, 3)
-    TestUtils.waitUntilMetadataIsPropagated(servers, topic3, 4)
-    TestUtils.waitUntilMetadataIsPropagated(servers, topic3, 5)
-    TestUtils.waitUntilMetadataIsPropagated(servers, topic3, 6)
+    TestUtils.waitForPartitionMetadata(servers, topic3, 1)
+    TestUtils.waitForPartitionMetadata(servers, topic3, 2)
+    TestUtils.waitForPartitionMetadata(servers, topic3, 3)
+    TestUtils.waitForPartitionMetadata(servers, topic3, 4)
+    TestUtils.waitForPartitionMetadata(servers, topic3, 5)
+    TestUtils.waitForPartitionMetadata(servers, topic3, 6)
```

Comment:
Don't replace them with `waitForAllPartitionsMetadata` because I'm afraid it'll break the original testing purposes. And same as other places.